docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+sharelatex:
+  image: sharelatex/sharelatex
+  links:
+    - mongo
+    - redis
+  ports:
+    - "80:80"
```

Comment:
Hi @MaxenceG2M ,
Sorry for my late reply.
Yes you are right - being consistent with the docs is a valid point.
Maybe I can find the time to update the pull request, just to have things documented somewhere ;) (Plus I need to figure out how to edit it - I have delete the repo quite some time ago)